[2023-05-26 05:58:51,683] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-31T00:00:00+00:00 [queued]>
[2023-05-26 05:58:51,693] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-31T00:00:00+00:00 [queued]>
[2023-05-26 05:58:51,694] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 05:58:51,695] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-05-26 05:58:51,696] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 05:58:51,709] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): db_connect> on 2023-01-31 00:00:00+00:00
[2023-05-26 05:58:51,714] {standard_task_runner.py:52} INFO - Started process 433 to run task
[2023-05-26 05:58:51,717] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tercera_entrega_ecentoz', 'db_connect', 'scheduled__2023-01-31T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/dag_tercera_entrega.py', '--cfg-path', '/tmp/tmpux450iln', '--error-file', '/tmp/tmp7v2ga0lp']
[2023-05-26 05:58:51,719] {standard_task_runner.py:80} INFO - Job 32: Subtask db_connect
[2023-05-26 05:58:51,782] {task_command.py:371} INFO - Running <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-31T00:00:00+00:00 [running]> on host f4aa0d8de4f3
[2023-05-26 05:58:51,845] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ecentoz
AIRFLOW_CTX_DAG_ID=tercera_entrega_ecentoz
AIRFLOW_CTX_TASK_ID=db_connect
AIRFLOW_CTX_EXECUTION_DATE=2023-01-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-31T00:00:00+00:00
[2023-05-26 05:59:13,315] {python.py:173} INFO - Done. Returned value was: None
[2023-05-26 05:59:13,392] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tercera_entrega_ecentoz, task_id=db_connect, execution_date=20230131T000000, start_date=20230526T055851, end_date=20230526T055913
[2023-05-26 05:59:13,429] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-26 05:59:13,458] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
